<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="upperFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TouchPiano" name="upperTouchPiano" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="lowerFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TouchPiano" name="lowerTouchPiano" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend_configuration"/>
   <addaction name="actionSettingsMenu"/>
   <addaction name="actionDeviceInfo"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionSettingsMenu">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/images/noto/emoji_u1f527.png</normaloff>:/images/noto/emoji_u1f527.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSend_configuration">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/images/noto/emoji_u1f680.png</normaloff>:/images/noto/emoji_u1f680.png</iconset>
   </property>
   <property name="text">
    <string>Send configuration</string>
   </property>
  </action>
  <action name="actionDeviceInfo">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/images/noto/emoji_u2139.png</normaloff>:/images/noto/emoji_u2139.png</iconset>
   </property>
   <property name="text">
    <string>Device information</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/images/noto/emoji_u1f4da.png</normaloff>:/images/noto/emoji_u1f4da.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TouchPiano</class>
   <extends>QWidget</extends>
   <header>touch_piano.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
